manifests/conf.pp

```diff
@@ -45,8 +45,8 @@
   Optional[Integer] $shredcycles                     = undef,
   Optional[Integer] $start                           = undef,
   Boolean $su                                        = false,
-  String $su_user                                    = 'root',
-  String $su_group                                   = 'root',
+  String $su_user                                    = $logrotate::root_user,
```

Comment:
The tests fail as I would expect here. 

There's no way  to know that the logrotate class has been included when this type is loaded and the
documentation says loading the class is optional.

Only way AFAIK is set the var to undef and then case the default based on facts if left undef.